examples/IFRS.py

```diff
@@ -10,7 +10,7 @@
 corporation = Ledger(residual_account_name='Retained earnings')
 bank = Ledger(residual_account_name='Retained earnings')
 
-corporation.make_stock_accounts([
+corporation.make_asset_accounts([
```

Comment:
@siebenbrunner @DavoudTaghawiNejad should `examples/IFRS.py` be added to the test case? Currently `make_asset_accounts`, `make_liability_accounts` are not yet covered by `tests/tdd.py` and `tests/banking.py`.

Reply:
yes